### ISSUE TITLE
Make pytest-playwright available on conda

### ISSUE THREAD

**Issue by maximlt** [microsoft/playwright-pytest#113]
`playwright` can be downloaded with conda from the [Microsoft channel](https://anaconda.org/microsoft/playwright). I'd be nice to have `pytest-playwright` also available on this channel.

And thanks for the great work, I love this pytest plugin!

**Comment by owenlamont** [comment]
I'd also love a conda packaged version of pytest-playwright. If there's no objections I'd be happy to try creating a package for this on conda-forge - although it probably makes more sense to come from the Microsoft channel if playwright itself is there.

**Comment by mxschmitt** [comment]
Yeah I feel like we should do the same to what we do inside the https://github.com/microsoft/playwright-python publishing wise. Happy to accept patches for it.

**Comment by mxschmitt** [comment]
Its available now thanks to @owenlamont

See here: https://anaconda.org/Microsoft/pytest-playwright

**Comment by owenlamont** [comment]
Thanks for finishing it off @mxschmitt - I hope it wasn't too painful resolving those  last deployment issues.
